internal/manifest/manifest.go

```diff
@@ -24,25 +24,25 @@ type Provider struct {
 }
 
 type HetznerDNS struct {
-	Name     string `validate:"required" yaml:"name"`
+	Name     string `validate:"required,max=15" yaml:"name"`
```

Comment:
Shouldn't we write in the [API reference](https://docs.claudie.io/latest/input-manifest/api-reference/#providers) that the name of the cloud provider is limited by 15 chars?